src/ccutil/ccutil.cpp

```diff
@@ -48,6 +49,12 @@ void CCUtil::main_setup(const std::string &argv0, const std::string &basename) {
 
   const char *tessdata_prefix = getenv("TESSDATA_PREFIX");
 
+  // Ignore TESSDATA_PREFIX if there is no matching filesystem entry.
+  if (tessdata_prefix != nullptr && ! std::filesystem::exists(tessdata_prefix)) {
```

Comment:
If this is not a space typo `! ` it is easier to use fs as
```
namespace fs = std::filesystem;
if (!fs::exists(...))
```

Reply:
It was a space typo which is fixed now, thanks for the hint.